bugbug/repository.py

```diff
@@ -20,6 +25,20 @@
 BUG_PATTERN = re.compile('[\t ]*[Bb][Uu][Gg][\t ]*([0-9]+)')
 
 
+def hg_log(repo_dir):
+    hg = hglib.open(repo_dir)
+    args = cmdbuilder(b'log', template='{rev}\\0{node}\\0{tags}\\0{branch}\\0{author}\\0{desc}\\0{date}\\0')
+    x = hg.rawcommand(args)
+    out = x.split(b('\0'))[:-1]
+    revs = []
+    for rev in util.grouper(7, out):
+        posixtime = float(rev[6].split(b('.'), 1)[0])
+        dt = datetime.fromtimestamp(posixtime)
+        revs.append(revision(rev[0], rev[1], rev[2], rev[3], rev[4], rev[5], dt))
```

Comment:
Just use a normal tuple, why returning a `revision` object?

Reply:
When you remove `revision`, you should also be able to remove the `commits = (tuple(commit) for commit in commits)` line.